Sources/SwiftParser/ExpressionInterpretedAsVersionTuple.swift

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(RawSyntax) import SwiftSyntax
+
+/// Parse the source code of this node as a `VersionTupleSyntax`.
+///
+/// In some situations, like the `_version` argument of the `canImport` condition in `#if`, the version gets parsed as a normal expression, which results in
+/// - an integer literal for single-component versions
+/// - a floating point number for two-component versions
+/// - a floating point number with a member accesses for versions with more than three components
+///
+/// This is done so the parser can parse `canImport` like any other function call, reducing the need for special handling.
+///
+/// This property re-interprets such an expression as a version tuple in cases where the client know that it should semantically represent a version.
```

Comment:
Could you add the comment on the `interpretedAsVersionTuple` member? Otherwise it won’t show up in docc or if you invoke Quick Help on the property.